### ISSUE TITLE
List of companies uisng plivoframework

### ISSUE THREAD

**Issue by estsauver** [plivo/plivoframework#141]
Hey Plivo, 

It would be cool to add a list of companies that are using this to the repo (or maybe the marketing website.) We'd love to use plivo.com to support the core development, but there isn't yet inbound number availability in Kenya, and it'd be great to get insight into companies that are using this in other markets. 

~Earl

**Comment by alexjurgensen** [comment]
I agree. Would be great to see who else is using this open-source code.
